src/doc/trpl/concurrency.md

```diff
@@ -0,0 +1,343 @@
+% Concurrency
+
+Concurrency and paralellism are incredibly important topics in computer
```

Comment:
Sigh, I thought I had spellcheck on, but I guess not. Thanks for all the saves here.